pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -2245,7 +2248,7 @@ private void createPendingLoadTopic() {
 
         final String topic = pendingTopic.getLeft();
         try {
-            checkTopicNsOwnership(topic);
+            checkTopicNsOwnership(topic).get();
```

Comment:
Do we need to add a timeout here?

Reply:
I don't think it is necessary. Because in https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/OwnershipCache.java#L267 already have timeout.